src/plots/cartesian/axes.js

```diff
@@ -1302,8 +1302,8 @@ axes.makeClipPaths = function(td) {
 //     'redraw' to force full redraw, and reset ax._r
```

Comment:
All the changes in this file are just linting that got added as one of the initial commits.